it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala

```diff
@@ -366,18 +366,21 @@ class PollingNewMessagesWithWorkItemJobItSpec extends PlaySpec
         movements.size mustBe 1
         assertResults(movements.find(_.consignorId.equals("1")).get, cachedMovement1)
 
-        val workItems = findAll().futureValue
-
-      val workItem = workItems.find(_.item.exciseNumber.equals("1")).get
-      workItem.status mustBe ProcessingStatus.ToDo
+        eventually {
+          verifyItemAvailableAt(
+            createdWorkItem.availableAt.plusSeconds(2 * 60).truncatedTo(ChronoUnit.MILLIS)
+          ).futureValue mustBe true
+        }
 
-      //Work Item repo truncates to Milliseconds
-      workItem.availableAt mustBe createdWorkItem.availableAt.plusSeconds(2 * 60).truncatedTo(ChronoUnit.MILLIS)
+        eventually {
+          verifyItemStatus(ProcessingStatus.ToDo).futureValue mustBe true
+        }
 
       }
     }
 
-    "if fails mark work item as failed so can be retried" in {
+    //todo: This test seems to be the same as the above one. Can it be removed?
```

Comment:
I think it was originally meant to from the title saying it should be failed but the implementation is the same. So need to either make this test actually test it failed - not sure how as the polling will retry it - or remove it.

Reply:
I will leave a todo to look at this test at later time.

Reply:
removed the duplicate test now